examples/user_interaction/client/src/App.js

```diff
@@ -29,12 +29,14 @@ class App extends React.Component {
   handleClick() {
     // Use promises to test behavior on MicroTasks.
     const promise = new Promise(resolve => {
+      console.log(window.Zone.current);
```

Comment:
Are these log statements meant to be temporary?

Reply:
Sorry, I was doing something different. Not related, just deleted this.